fonts/README.md

```
@@ -0,0 +1,83 @@
+@import '@osvauld/fonts/src/fonts/plus-jakarta-sans.css';
+```
+
+### In TypeScript/JavaScript
+```typescript
+import '@osvauld/fonts/src/fonts/inter.css';
+import { fontFamilies } from '@osvauld/fonts';
+```
+
+## Font Files Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)